src/Stratis.SidechainD/Program.cs

```diff
@@ -70,6 +66,35 @@ public static async Task MainAsync(string[] args)
             }
         }
 
+        private static IFullNode GetApexFullNode(NodeSettings nodeSettings)

```

Comment:
Do we even need to keep Apex anywhere?

Reply:
Sentimentally attached, but I don't mind deleting it